controllers/widget.js

```diff
@@ -104,6 +104,19 @@ function applyProperties(properties) {
     _applyInnerProperties(_properties);
 }
 
+function setAccessibilityLabel(label, hint) {
+
+    $.inner.setAccessibilityLabel(label);
+    $.outer.setAccessibilityLabel(label);
```

Comment:
Thanks @nicolasbd. Won't it causes issues if you add it to both? I could see audio feedback repeating the same thing twice?